src/reports.py

```diff
@@ -0,0 +1,112 @@
+import json
+import logging
+import os
+from datetime import datetime, timedelta
+from functools import wraps
+from typing import Any, Optional, TypeVar
+
+import pandas as pd
+from black.lines import Callable
```

Comment:
Не правильный импорт
```suggestion
from typing import Callable
```